### PR TITLE
Update it_it.json

### PR DESCRIPTION
**Pull request by TheBossMagnus**
1) Sorry a bit late
2) I used minecraft italian translation of the name of blocks, but generally in Italy the English name is used. What are your guidelines?
3) Do i have to pr this to all other branches?

### CODE REVIEW

File: src/main/resources/assets/bettergrass/lang/it_it.json [hunk 1]
```diff
@@ -2,21 +2,21 @@
   "bettergrass.title": "BetterGrassify Config",
```
Comment (UltimatChamp) [inline src/main/resources/assets/bettergrass/lang/it_it.json:2]: Can you translate `Config` here?

**Review by UltimatChamp — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/bettergrass/lang/it_it.json (hunk 1 above)
Comment (TheBossMagnus) [reply src/main/resources/assets/bettergrass/lang/it_it.json:2]: Idk if it sounds better translated or not, but as you wish

File: src/main/resources/assets/bettergrass/lang/it_it.json [hunk 2]
```diff
@@ -2,21 +2,21 @@
...
-  "bettergrass.betterSnowMode.optifine": "OptiFine-like",
-  "bettergrass.betterSnowMode.lambda": "LambdaBetterGrass-like",
-  "bettergrass.excludedTags": "Excluded Tags",
-  "bettergrass.excludedBlocks": "Excluded Blocks",
-  "bettergrass.noconfig.warn": "§cBetterGrassify for (Neo)Forge 1.20.1 is unable to display a configuration screen due to a limitation in Forge. Please use the configuration file instead. Sorry for the inconvenience!§r\n\nIf you're using Embeddium, you can change most of the configurations in the \"Video Settings Menu.\""
+  "bettergrass.betterSnow": "Neve Migliorata",
+  "bettergrass.betterSnowMode": "Modalità Neve Migliorata",
+  "bettergrass.betterSnowMode.desc": "§4Off §r- Non verrà generato alcun livello di neve.\n§3Simile a OptiFine §r- Mostra uno strato di neve sotto blocchi specifici che hanno almeno uno strato di neve o un blocco di neve vicino a loro.\n§dSimile a LambdaBetterGrass § r- Mostra uno strato di neve sotto blocchi specifici che hanno almeno \"due\" strati di neve o blocchi di neve vicino a loro.",
+  "bettergrass.betterSnowMode.optifine": "Come OptiFine",
```
[the remark this answers is not in the document]
Comment (TheBossMagnus) [reply src/main/resources/assets/bettergrass/lang/it_it.json:17]: It's the same, maybe the first one is clearer

**Comment by UltimatChamp** [comment]
Thanks!

**Pull request merged**